clients/src/test/java/org/apache/kafka/common/network/SslSelectorTest.java

```diff
@@ -156,6 +161,85 @@ public void testDisabledRenegotiation() throws Exception {
 
     }
 
+    @Override
+    public void testMuteOnOOM() throws Exception {
+        //clean up default selector, replace it with one that uses a finite mem pool
+        selector.close();
+        MemoryPool pool = new SimpleMemoryPool(900, 900, false);
+        //the initial channel builder is for clients, we need a server one
+        File trustStoreFile = File.createTempFile("truststore", ".jks");
+        Map<String, Object> sslServerConfigs = TestSslUtils.createSslConfig(false, true, Mode.SERVER, trustStoreFile, "server");
+        sslServerConfigs.put(SslConfigs.PRINCIPAL_BUILDER_CLASS_CONFIG, Class.forName(SslConfigs.DEFAULT_PRINCIPAL_BUILDER_CLASS));
+        sslClientConfigs = TestSslUtils.createSslConfig(false, false, Mode.SERVER, trustStoreFile, "server");
+        channelBuilder = new SslChannelBuilder(Mode.SERVER);
+        channelBuilder.configure(sslClientConfigs);
```

Comment:
Minor typo (doesn't impact the test, but is confusing). I think you want to use `sslServerConfigs` here and remove `sslClientConfigs` setting just above since only one server channel builder is used in the test?